client/src/components/relatedProducts/ProductCard.jsx

```diff
@@ -3,26 +3,27 @@ import Comparison from './Comparison.jsx'
 import { AiOutlineStar } from 'react-icons/ai'
 import helper from '../../helpers.js';
 
-const ProductCard = ({ product }) => (
+import { useParams } from 'react-router-dom'
 
-  <div className='product-card'>
-    <div className='card-container'>
-      <div className='card-visuals'>
-        <Comparison />
-        <div className='card-image'>
-          <img src={product.photo} />
+export default function ProductCard(props) {
+
+  return (
+    < div className='product-card' >
+      <div className='card-container'>
+        <div className='card-visuals'>
+          <Comparison />
+          <div className='card-image' onClick={() => window.location.pathname = `/product/${props.product.id}/`}>
```

Comment:
Sorry if there is a more elegant way to do it. Was trying to use Link from react-router but got stuck

Reply:
Link actually causes some problems here since we want to click the card not a button. Something else you might look into is the hook useHistory. We still leverage react router here by pushing a route onto the history during the onClick.

Reply:
Will look into useHistory post-deployment